kiwi/xml_description.py

```diff
@@ -52,9 +54,9 @@ class XMLDescription(object):
         path to base XML description file
 
     """
-    def __init__(self, description, derived_from=None):
+    def __init__(self, description=None, derived_from=None, content=None):
         self.description_xslt_processed = NamedTemporaryFile()
-        self.description = description
+        self.description = content and BytesIO(bytes(content)) or description
```

Comment:
@isbm I am sorry if I am upsetting you, not my intention.

Code is to be read so others can understand it. You are submitting the code, and I am this 'other' that is trying to understand it, and to do that I am making some reasonable and honest technical questions. You can decide to answer it or not, but in any case, please, avoid this kind of bullshit.